source/code-snippets/usage-examples/findOne.js

```diff
@@ -0,0 +1,22 @@
+
+const { MongoClient } = require('mongodb');
+
+
+const client = new MongoClient(
+   'mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority',
+);
+
+async function run() {
+   try {
+      await client.connect();
+      const database = client.db('sample_mflix');
+      const collection = database.collection('movies');
+      const query = { title: 'The Room' };
+      const movie = await collection.findOne(query);
+      // since this method returns the matched document, not a cursor, print it directly
+      console.log(movie);
+   } finally {
```

Comment:
IMO we should demonstrate how to handle an error as close to the throw as possible. That would be either:

- `try { ... } catch (err) { console.error(err) } finally { ... }`
- `findOne(query).catch(err => console.error(err))`

We should probably also have a code comment or other info on what could cause an error and how to respond to one.